introduction.md

```diff
@@ -1,3 +1,9 @@
 My name is Hatem Saidane.
 Currently learning about Master GitHub
 My favorite code school path is GitHub and JavaScript.
+My Latest Skills
+* Git
+* C++
+* Delphi
+* JavaScrtips
+* HTML
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?